README.md

```diff
@@ -421,6 +425,26 @@ func main() {
 }
```

Comment:
Maybe we should remove `func main()` from here so the examples are consistent?

Reply:
@fotos I've added `func main() {` to other places instead